api/v1beta1/groupversion_info.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
```

Comment:
Why this change?

We shouldn't change years in files, many occurrences in this PR

Reply:
all resolved now. Confirmed each file by viewing the commit history of the individual files

Reply:
I am still seeing change of year from 2022 to 2023